server/routes/api/shares/shares.ts

```diff
@@ -152,7 +152,28 @@ router.post(
         offset: ctx.state.pagination.offset,
         limit: ctx.state.pagination.limit,
       }),
-      Share.count({ where }),
+      Share.count({
```

Comment:
Tiny bug with shares count which was not in parity with the actual available data.
For admins, we did not check if they actually had access to the shared doc (ex: doc in another user's private collection).